packages/components/styles/_forms-variables.scss

```diff
@@ -0,0 +1,11 @@
+// General purpose tokens
+$dt-color-form-border-color: $color-gray-500;
+$dt-color-form-border-color-focus: $color-blue-500;
```

Comment:
Probably don't need to tackle in this PR but we could potentially create a more global stylesheet to store focus ring variables since they shouldn't differ from component to component

Reply:
Indeed a good one, pop a ticket in the backlog